src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random rm = new Random();
+
     public String getRandomColor() {
-        return null;
+        Colors[] values = Colors.values();
+        int length = values.length;
+        int randIndex = new Random().nextInt(length);
```

Comment:
no need to create an instance of `class Random` each time #getRandomColor is being invoked. 

use the `rm` field that you have instead.